.travis.yml

```diff
@@ -8,17 +8,17 @@ language: python
 
 matrix:
     include:
-        - python: 2.7
+        - python: 3.6
           os: linux
-          env: TOXENV=py27
-        - python: 2.7
+          env: TOXENV=py36
+        - python: 3.6
           os: linux
           env: TOXENV=pylama
         - language: generic
           os: osx
-          # 7.2 is OS X 10.11.x
+          # 10.1 is OS X 10.13.x
           # https://docs.travis-ci.com/user/languages/objective-c/#Supported-OS-X-iOS-SDK-versions
-          osx_image: xcode7.2
+          osx_image: xcode10.1
```

Comment:
Again. We want both py27 and py36 for Linux and Windows. Means you cannot simply remove the py27 entries.

Reply:
Ugh, got it !
I will add an new update. Thanks !

Reply:
But as I see, there is only Linux with Python 2.7 currently.
Do you want me to add also Windows run ?

Reply:
Windows is covered by Appveyor given that Travis CI only supports Linux and MacOS.

Reply:
And the .appveyor config looks all fine and results are all green: https://ci.appveyor.com/project/whimboo/mozdownload/builds/22544085

Reply:
Got it !